.env

```diff
@@ -1 +1,3 @@
-REACT_APP_GOOGLE_MAPS_API_KEY = ''
\ No newline at end of file
+REACT_APP_GOOGLE_MAPS_API_KEY = ''
+REACT_APP_SELLER_APP_ID=''
```

Comment:
Still here ?

Reply:
Seller App ID is needed for BPP context builder in Frontend for /select

Reply:
Updated